### PR TITLE
Ivan/ch8157/build clubhouse js client library

### PR DESCRIPTION
**Pull request by camilleacey**

### CODE REVIEW

File: tutorials/importing_data.md [hunk 1]
```diff
@@ -1,89 +1,47 @@
...
-```
+#### Stories
 
-Each method in clubhouse.js returns a
-[bluebird](http://bluebirdjs.com/docs/getting-started.html) `Promise`
-object. That means you can use any of the combinator functions provided by
-the `bluebird` library. For example if you wanted to filter all of the projects that name
-start with the word `Mobile` you would do the following,
+[Stories](https://clubhouse.zendesk.com/hc/en-us/articles/205587589-The-Stories-Page-and-The-Story-Card) are the basic unit of work in Clubhouse
+At minimum, Stories must have title and a Requester (this defaults to the user who created the token being used) and be assigned to a Project.
```
Comment (iwillig) [inline tutorials/importing_data.md]: Story Name

File: tutorials/importing_data.md [hunk 2]
```diff
@@ -1,89 +1,47 @@
...
+var story = {
+    'name': 'This is a test story',
+    'project_id': 274
+};
 
-```javascript
-clubhouse.listProjects(function (error, projects) {
-    if (error) { throw error };
-    console.log(projects);
+clubhouse.createStory(story).then(function (story) {
```
Comment (iwillig) [inline tutorials/importing_data.md]: `newStory`

File: tutorials/importing_data.md [hunk 3]
```diff
@@ -94,10 +52,9 @@ clubhouse.createProject(project).then(function (project) {
...
-
-You can also create a new story as a part of an epic
-[epic](https://clubhouse.zendesk.com/hc/en-us/articles/205274149-Epics-and-The-Epics-Page).
-This is optional and a story does not need to be in an epic.
+You can also assign a new Story to an
+[Epic](https://clubhouse.zendesk.com/hc/en-us/articles/205274149-Epics-and-The-Epics-Page).
+This is optional and a Story does not need to be assigned to an Epic.
 
 ```javascript
 var epic = { 'name': 'This is a test epic' };
```
Comment (iwillig) [inline tutorials/importing_data.md]: Don't use `new`

File: tutorials/importing_data.md [hunk 4]
```diff
@@ -403,8 +417,8 @@ Promise.props({
 });
 ```
 
-`Promise.props` is useful for builing an object that has the define
-properties that are the resulting of each Promise object.
+`Promise.props` is useful for building an object that has defined
+properties from the results of each Promise object.
```
Comment (iwillig) [inline tutorials/importing_data.md]: Promise.props creates a object that matches the properties you defined.

File: tutorials/importing_data.md (hunk 3 above)
Comment (camilleacey) [inline tutorials/importing_data.md]: I don't remember what this referred to...

**Comment by whitchapman** [comment]
[ch8157]

**Pull request merged**